frontend/dockerfile/linter/generate.go

```diff
@@ -35,6 +36,11 @@ aliases:
   - {{ .Rule.URLAlias }}
 {{- end }}
 ---
+{{- if .Rule.Experimental }}
+> **Note**
+>
+> This check is experimental.
+{{- end }}
```

Comment:
```suggestion
{{- if .Rule.Experimental }}

> **Note**
>
> This check is experimental and is not enabled by default. To enable it, see
> [Experimental checks](https://docs.docker.com/go/build-checks-experimental/).
{{- end }}
```

- needs: https://github.com/docker/docs/pull/20564

Reply:
Updated `_index.md` generator, updated the experimental text to include the link, and added a note to the docs pr. :)